slick/src/main/scala/akka/stream/alpakka/slick/javadsl/Slick.scala

```diff
@@ -105,10 +104,11 @@ object Slick {
    */
   def flowWithPassThrough[T, R](
       session: SlickSession,
+      executorService: ExecutorService,
```

Comment:
I think it is fine to use the Scala API here, if Java users wants to provide an ´ExecutorService` they can use `akka.dispatch.ExecutionContexts.fromExecutorService`. Most common use case will be to use the one the dispatcher.

Reply:
Ok. I'm not familiar with using Akka with Java.
What about replacing it with `dispatcher: ExecutionContextExecutor`? Then the `ActorSystem.dispatcher` can be used. Although the type is `ExecutionContextExecutor`, I thought it might be better to name it `dispatcher` just like in `ActorSystem`.

Reply:
Oh I see I was maybe a tad unclear there, accept an `ExecutionContext` is fine is what I meant. Usually we just call such params `executionContext` (or `ec` when it's an implicit) even if it will most likely be the dispatcher.